docker-for-windows/faqs.md

```diff
@@ -10,55 +10,41 @@ for knowledge base articles, FAQs, technical support for subscription levels, an
 
 * The **Stable channel** provides a general availability release-ready installer
   for a fully baked and tested, more reliable app. The Stable version of Docker
-  Desktop for Windows comes with the latest released version of Docker Engine. The
+  Desktop includes the latest released version of Docker Engine. The
   release schedule is synced with Docker Engine releases and hotfixes. On the
   Stable channel, you can select whether to send usage statistics and other data.
 
-* The **Edge channel** provides an installer with new features we are working on,
-  but is not necessarily fully tested. It comes with the experimental version of
-  Docker Engine. Bugs, crashes, and issues are more likely to occur with the Edge
-  app, but you get a chance to preview new functionality, experiment, and provide
-  feedback as the apps evolve. Releases are typically more frequent than for
-  Stable, often one or more per month. Usage statistics and crash reports are
-  sent by default. You do not have the option to disable this on the Edge channel.
+* The **Edge channel** provides an installer with new features we are working on, but is not necessarily fully tested. It includes the experimental version of
+  Docker Engine. Bugs, crashes, and issues can occur when using the Edge version, but you get a chance to preview new functionality, experiment, and provide feedback as the apps evolve. Edge releases are typically more frequent than for Stable, often one or more per month. Usage statistics and crash reports are sent by default. You do not have the option to disable this on the Edge channel.
 
-### Can I switch back and forth between Stable and Edge versions of Docker Desktop for Windows?
+### Can I switch back and forth between Stable and Edge versions of Docker Desktop?
 
 Yes, you can switch between versions to try out the Edge release to see what's
 new, then go back to Stable for other work. However, **you can have only one app
 installed at a time**. Switching back and forth between Stable and Edge apps can
 destabilize your development environment, particularly in cases where you switch
-from a newer (Edge) channel to older (Stable).
+from a newer (Edge) channel to older version (Stable).
 
-For example, containers created with a newer Edge version of Docker Desktop for Windows
-may not work after you switch back to Stable because they may have been created
-leveraging Edge features that aren't in Stable yet. Just keep this in mind as
-you create and work with Edge containers, perhaps in the spirit of a playground
-space where you are prepared to troubleshoot or start over.
+For example, containers created with a newer Edge version of Docker Desktop may not work after you switch back to Stable because they may have been created leveraging Edge features that aren't in Stable yet. Just keep this in mind as you create and work with Edge containers, perhaps in the spirit of a playground space where you are prepared to troubleshoot or start over.
```

Comment:
Should we give more information here? I wonder if users may get confused on how new or what's the latest version they should have? Also, suggest changing "...leveraging Edge features..." to "...using Edge features...".

Reply:
Users can choose between Edge and Stable releases. Information about what's included in each new release is documented in the release notes. There's also a caveat in the Edge release notes that some of the features may not make it to the Stable release. 

Updated leveraging to using in commit 59aa7e5.